sdcm/utils/raft/common.py

```diff
@@ -50,6 +51,7 @@ def validate_raft_on_nodes(nodes: list[BaseNode]) -> None:
     LOGGER.debug("Raft is ready!")
 
 
+@retrying(allowed_exceptions=(RaftTopologyCoordinatorNotFound, ), message="Waiting topology coordinator election ...")
```

Comment:
I would recommend defining the number of retries, for clarity

Reply:
fixed